src/shell_utils.py

```diff
@@ -0,0 +1,31 @@
+# src/shell_utils.py
+import os
+from src.ssh_client import SSHClient
+from src.winrm_client import WinRMClient
+
+
+def get_script_paths(shell_client, base_dir, full_directory_path):
+    if isinstance(shell_client, SSHClient):
+        local_script = os.path.join(base_dir, '../scripts/create_dir.sh')
+        remote_script = '/tmp/create_dir.sh'
```

Comment:
Remote script path should be fetched dynamically ( returned by upload_file function)